utils/githooks/pre-commit.d/50-clang-format.sh

```diff
@@ -13,11 +14,11 @@ if [ -e .git/MERGE_HEAD ]; then
 fi
 
 if ! command -v git-clang-format > /dev/null 2>&1; then
-    echo "git-clang-format not installed. Skipping"
+    echo "git-clang-format not installed. See ./utils/githooks/README.md for instructions"
     exit 0
 fi
 if ! command -v clang-format > /dev/null 2>&1; then
-    echo "clang-format not installed. Skipping"
+    echo "clang-format not installed. See ./utils/githooks/README.md for instructions"
```

Comment:
`./utils/githooks/README.md#2-install-the-required-tools`
Let's try to use link to particular section - helps a lot.